code/datums/diseases/Cancer.dm

```diff
@@ -0,0 +1,61 @@
+/datum/disease/Cancer
+	form = "Condition"
+	name = "Space Cancer"
+	max_stages = 5
+	cure_text = "Surgery"
+	agent = "Cancer Cells"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 1
+	desc = "The Old and Trusty Cancer but in SPACE."
+	severity = "Very Dangerous!"
+	disease_flags = CAN_CARRY|CAN_RESIST
+	spread_flags = NON_CONTAGIOUS
+	visibility_flags = HIDDEN_PANDEMIC
+	required_organs = list(/obj/item/organ/internal/Lungs)
+	bypasses_immunity = TRUE
+	virus_heal_resistant = TRUE
+
+/datum/disease/Cancer/stage_act()
+	..()
+	switch(stage)
+		if(1)
+			if(prob(5))
+				affected_mob.emote("cough")
+		if(2)
+			var/obj/item/organ/internal/Lungs/A = affected_mob.get_int_organ(/obj/item/organ/internal/Lungs)
+			if(A)
+				A.inflamed = 1
```

Comment:
so you literally copy-pasted appendix code, then didn't include it in the .dme

Yeah, no, this is not going to pass--include it, and test your stuff, please.